html/semantics/forms/the-form-element/form-requestsubmit.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<iframe name="iframe" src="about:blank"></iframe>
+
+<script>
+test(() => {
+  document.body.insertAdjacentHTML('afterbegin', '<form><input type="reset"></form>');
+  let form = document.querySelector('form');
+  assert_throws(new TypeError(), () => {
+    form.requestSubmit(document.body);
+  });
+  assert_throws(new TypeError(), () => {
+    form.requestSubmit(form.querySelector('input[type=reset]'));
+  });
+}, 'Passing an element which is not a submit button should throw');
```

Comment:
A reset button is also not a submittable element. Also test a submittable element that is not a submit button (like `input type=text`)

Reply:
Done.